core-services/prow/02_config/generate-boskos.py

```diff
@@ -199,6 +199,9 @@
     'alibabacloud-cn-qe-quota-slice': {
         'us-east-1': 10,
     },
+    'hypershift-hive-quota-slice': {
+        'default': 80
```

Comment:
Where do we get the number, 80?

Reply:
Consulted with HyperShift team. This is the limit for the number of hosted cluster supported on one management cluster